server/vulnerabilities/io/github.go

```diff
@@ -63,7 +65,30 @@ func (gh GitHubClient) Download(URL string) (string, error) {
 }
 
 // Bulletins returns a map of 'bulletin name' => 'download URL' of the bulletins stored as assets on Github.
-func (gh GitHubClient) Bulletins(ctx context.Context) (map[SecurityBulletinName]string, error) {
+func (gh GitHubClient) MSRCBulletins(ctx context.Context) (map[MetadataFileName]string, error) {
+	return gh.list(ctx, mSRCFilePrefix, NewMSRCMetadata)
+}
+
+func (gh GitHubClient) MacOfficeReleaseNotes(ctx context.Context) (MetadataFileName, string, error) {
+	resultMap, err := gh.list(ctx, macOfficeReleaseNotesPrefix, NewMacOfficeRelNotesMetadata)
+	if err != nil {
+		return MetadataFileName{}, "", err
+	}
+
+	// We should only have a single release notes metadata file on GH ....
+	if len(resultMap) > 1 {
+		return MetadataFileName{}, "", errors.New("found more than one MacOffice release notes")
+	}
+
+	for k, v := range resultMap {
+		return k, v, nil
```

Comment:
I may have missed something here.
But will this not return the first found item in resultMap? Is that the intension?

Reply:
Yes, there should only be a single Mac Office release note JSON file on Github, so we just return the first one - I'll add a comment explaining this.

Reply:
Understood. Thanks!